decidim-proposals/app/cells/decidim/proposals/highlighted_proposals_for_component/show.erb

```diff
@@ -11,7 +11,7 @@
   <% end %>
 </div>
 
-<div class="flex items-start justify-between">
+<div class="flex items-center justify-between p-2">
```

Comment:
`p-2` will add padding everywhere, and broke padding. use only vertical padding, if really needed.